### PR TITLE
New package: Openstreetmap.Osm2pgsql version 1.7.2

### PR DESCRIPTION
**Pull request by noenandre**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94824)

### CODE REVIEW

**Comment by noenandre** [comment]
This is a version 1.4.0 manifest. Awaiting zip support.

**Comment by stephengillie** [comment]
Validation got an HTTP error:
> Exception Message: The remote server returned an error: (503) Server Unavailable. (type WebException)

File: manifests/o/OpenStreetMap/Osm2pgsql/1.7.2/OpenStreetMap.Osm2pgsql.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: OpenStreetMap.Osm2pgsql
+PackageVersion: 1.7.2
+InstallerLocale: en-US
+Commands:
+- osm2pgsql
+ReleaseDate: 2022-11-10
+Installers:
```
Comment (stephengillie) [inline manifests/o/OpenStreetMap/Osm2pgsql/1.7.2/OpenStreetMap.Osm2pgsql.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/o/OpenStreetMap/Osm2pgsql/1.7.2/OpenStreetMap.Osm2pgsql.installer.yaml:10]: The program installs normally, but errors when launched through Windows Explorer: 
image

When launched through PowerShell, it appears to do nothing. And with the dependency, launching through Windows Explorer also appears to do nothing. Is this the expected behavior for this package?

Comment (noenandre) [reply manifests/o/OpenStreetMap/Osm2pgsql/1.7.2/OpenStreetMap.Osm2pgsql.installer.yaml:10]: Hi! Yes, that is a expected behavior. osm2psql is a CLI tool. 
Visual C++ Redistributable is not mention as a dependency in the [installation instructions](https://osm2pgsql.org/doc/install.html#installing-on-windows), and it was of already installed on the machine this manifest was tested on.

**Comment by stephengillie** [comment]
Hi @noenandre,

This application doesn't seem to stay running, and it gives an error exit code. Is this expected? 

> Launching executable file: C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\osm2pgsql.exe
> Executable ran for 10 seconds: False
> Executable returned exit code: -1073741515

**Comment by noenandre** [comment]
No, seems like this PR has to wait on [#2711](https://github.com/microsoft/winget-cli/issues/2711)

**Pull request closed without merging**